app/_locales/en/messages.json

```diff
@@ -3352,7 +3352,7 @@
     "message": "Operation Failed"
   },
   "optimismFees": {
-    "message": "Optimism fees"
+    "message": "Layer 2 fees"
```

Comment:
Should we rename the key to layer2Fees?

Reply:
Done. I was playing it safe as I didn't want to find and replace every instance in the `messages.json` files but it seems this was never translated so only a single entry.